src/main/resources/application.yml

```diff
@@ -48,10 +48,13 @@ case:
     case-service: http://localhost:8082
     workflow-service: http://localhost:8091
     document-service: http://localhost:8083
+    info-service: http://localhost:8085
     identity:
       group:
       team:
       user:
+    migration:
+      parent-topic-uuid: 'eb939efd-8aa8-46e0-a9c3-9ae979e151ab'
```

Comment:
The rest of these are within a secret/ config map for the service, this probably should reside in the same. Whilst not secret, this is also migration specific. If you need this for local, chuck it within the application-local[-migration] configuration.